src/client/pages/AssessmentHome/PanEuropeanHome/hooks/useSections.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+import { Users } from '@meta/user'
+
+import { useCycle } from '@client/store/assessment'
+import { useUser } from '@client/store/user'
+import { useCountryIso } from '@client/hooks'
+import Collaborators from '@client/pages/AssessmentHome/FraHome/Collaborators'
+import RecentActivity from '@client/pages/AssessmentHome/FraHome/RecentActivity'
+
+enum SectionNames {
+  overview = 'overview',
+  userManagement = 'userManagement',
+  recentActivity = 'recentActivity',
+}
+
+type Section = {
+  name: SectionNames
+  component: React.FC
+}
+
+const Placeholder: React.FC = () => {
+  return <div />
+}
+
+export const useSections = (): Array<Section> => {
+  const user = useUser()
+  const countryIso = useCountryIso()
+  const cycle = useCycle()
+
+  const sections: Array<Section> = [{ name: SectionNames.overview, component: Placeholder }]
```

Comment:
the overview component must be created and includes:
<img width="1078" alt="Screenshot 2022-12-13 at 12 24 06" src="https://user-images.githubusercontent.com/1138156/207305276-84b67864-64c3-409d-8645-75f7f197672f.png">